tasks/copy_to_glacier/README.md

```diff
@@ -11,7 +15,7 @@ You are able to specify a list of file types (extensions) that you'd like to exc
 {
   ...
   "meta": {
-    "exclue_file_type": [".cmr", ".xml", ".cmr.xml"]
+    "exclude_file_type": [".cmr", ".xml", ".cmr.xml"]
```

Comment:
This needs to be `excludeFileTypes` instead of `exclude_file_type`.

Reply:
Updated to excludeFileTypes.